Cubical/Foundations/Equiv/Properties.agda

```diff
@@ -44,30 +56,18 @@ equivAdjointEquiv e = compEquiv (congEquiv e) (compPathrEquiv (retEq e _))
 invEq≡→equivFun≡ : (e : A ≃ B) → ∀ {a b} → invEq e b ≡ a → equivFun e a ≡ b
 invEq≡→equivFun≡ e = equivAdjointEquiv e .fst ∘ sym
 
-isEquivPreComp : {A B : Type ℓ} {C : Type ℓ′} (e : A ≃ B)
-               → isEquiv (λ (φ : B → C) → φ ∘ e .fst)
-isEquivPreComp {A = A} {B} {C} e = EquivJ
-                  (λ (A : Type _) (e' : A ≃ B) → isEquiv (λ (φ : B → C) → φ ∘ e' .fst))
-                  (idIsEquiv (B → C)) e
+isEquivPreComp : (e : A ≃ B)
+               → isEquiv (λ (φ : B → C) → φ ∘ equivFun e)
+isEquivPreComp e = snd (equiv→ (invEquiv e) (idEquiv _))
```

Comment:
Nice! So this lemma doesn't use univalence anymore now?

Reply:
With the lovely side effect of being more universe polymorphic, yes

Reply:
Wonderful!